packages/agents/backend/poller/src/lib/operations/transfers.ts

```diff
@@ -90,9 +90,11 @@ export const updateTransfers = async () => {
 
     if (xcalledTransfers.concat(executedTransfers).concat(reconciledTransfers).length == 0) {
       done = true;
+      logger.debug("Processed all pending transfers. Last page", requestContext, methodContext, { page });
```

Comment:
I think that would be better to have pageSize as well here. so like this {page, pageSize}

Reply:
Makes sense. Done